static/css/style.css

```diff
@@ -127,28 +127,25 @@ img{
 	-moz-columns: 2;
 }
 
-/* to hide title in the second column of Members*/
-.transparent{
-	color: transparent;
-}
 
 /* paragraph*/
-.intro{
+.intro {
 	text-align: justify;
 	color: #5C5C5C;
-	font-size: 20px;
+	font-size: 1px;
```

Comment:
you are right I was experimenting and I do not know why this is not working. I cannot change the size of the main content...